gossip.asciidoc

```diff
@@ -644,68 +644,388 @@ channel graph) has the following attributes:
+`query_short_channel_ids` message:
+
+  * `chain_hash`: The chain hash that denotes the genesis hash of the chain
+    we're querying for channels within.
+
+  * `len`: The number of bytes in the remaining payload.
+
+  * `encoded_short_channel_ids`: The set of encoded short channel IDs.
+
+  * `query_short_channel_ids_tlvs`: A set of optional TLV fields that are used
+    to extend the query system.
+
+Similar to the prior message in structure, this allows the syncing node toe
+request that the responding node send over the full information of the
+referenced short channel IDs, as they're missing from its local database.
+
+Upon receiving this message, the responding node should send over all
+`channel_announcement`, `channel_update`, and `node_announcement` messages (in
+that order) that are related to the reference short channel IDs. Note that the
+responding node can de-duplicate the `node_announcement` messages sent over in
+order to reduce redundant bandwidth usage.
+
+The `reply_short_channel_ids_end` message is sent once the responding node has
+sent over all the requested information:
+
+  * `chain_hash`: The chain hash that denotes the genesis hash of the chain
+    we're querying for channels within.
+
+  * `full_information`: A byte that denotes if the sending node has all
+    information about this block range or not. For example, light clients may
+    not have full information.
+
+Once this message is received, the syncing node will now increment its cursor,
+and start the scan phase again, until either it has all the channels it wants,
+or arrives at the current end of the main chain.
+
+At this point, our new node is able to sync the historical portion of the
+channel graph, and validate it entirely. At this point, we now enter the
+passive sync phase, rather than sync the new entries based on block height, we
+can opt to instead receive the set of updates as they stream in in real-time. 
+
+## Ongoing Channel Graph Maintenance
 
-### Channel Announcements
+At this pint, we're able to sync the channel graph based on the block heights
+that each of the channels were mined in. Once we have the full channel graph,
+we're able to use it to make decision w.r.t which channels to open, or for path
+finding to send payments. As mentioned earlier, the `channel_update` is a key
+component of the system as it allows routing nodes to update their routing
+policies due to changes in the market, chain conditions, or simply their own
+preference.
 
-### Purpose
-### Structure
-### Validation
+Once we're fully send the graph, we can now being to receive any new node
+announcements, channel announcements, or channel updates via the _gossip
+network_. By gossip network we simply mean a set of peers that rumor updates to
+the Channel Graph, in order to _eventually_ propagate their updates to the
+entire network. 
 
-### Channel Updates
+It's important to note that peers on the network _won't_ immediately being to
```

Comment:
why not spam protection or as a result of the timstamp_filter mechanism that can be used as discussed in the last paragraph